test/StoragelessTest/Http/SessionMiddlewareTest.php

```diff
@@ -124,6 +124,7 @@ public function testInjectsSessionInResponseCookies(callable $middlewareFactory)
         $token = $this->getCookie($response)->getValue();
 
         self::assertIsString($token);
+        self::assertTrue($token !== '');
```

Comment:
I would have loved to write this as `self::assertNotSame('', $token);` but unfortunately that specific assertion lacks the `@psalm-assert` spec :cry: https://github.com/sebastianbergmann/phpunit/blob/e733b040fd42101e6805ead23221db01ca39e259/src/Framework/Assert.php#L1387

Reply:
Eh, at least it's a test only 👍